### PR TITLE
Documentation link cleanup

### PR DESCRIPTION
**Pull request by bardiharborow**
This is a v4 rebase of #20283 with several other broken links fixed.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -37,11 +37,6 @@ <h2 class="bd-featurette-title">Easy to get started.</h2>
 {% endhighlight %}
       </div>
     </div>
-
-    <hr class="half-rule m-t-0">
-
-    <p><strong>Looking for something simpler?</strong> Customized builds of Bootstrap's CSS and JS are also available.</p>
-    <a href="{{ site.baseurl }}/getting-started/download/#custom-builds" class="btn btn-bs btn-outline">More download options</a>
```
Comment (mdo) [inline docs/index.html]: These lines shouldn't have been removed.

File: docs/index.html [hunk 2]
```diff
@@ -21,14 +21,14 @@ <h2 class="bd-featurette-title">Easy to get started.</h2>
...
     <div class="row">
-      <div class="col-sm-6 m-b-3">
+      <div class="col-sm-6">
         <h4>Managed dependencies</h4>
         <p>Include Bootstrap's source Sass and JavaScript files via Bower, Composer, Meteor, or npm. Package managed installs don't include documentation, but do include our Gruntfile and readme.</p>
         <p>
           <a class="btn btn-bs btn-outline" href="{{ site.baseurl }}/getting-started/download/#package-managers">View install methods</a>
         </p>
       </div>
-      <div class="col-sm-6 m-b-3">
```
Comment (mdo) [inline docs/index.html]: Why remove the margins?

File: docs/components/forms.md [hunk 3]
```diff
@@ -339,7 +339,7 @@ Assistive technologies such as screen readers will have trouble with your forms
 
 ### Using the Grid
 
-For more structured form layouts that are also responsive, you can utilize Bootstrap's [predefined grid classes](/layout/grid/#predefined-classes) or [mixins](/layout/grid/#sass-mixins) to create horizontal forms. Add the `.row` class to form groups and use the `.col-*-*` classes to specify the width of your labels and controls.
+For more structured form layouts that are also responsive, you can utilize Bootstrap's [predefined grid classes]({{ site.baseurl }}/layout/grid/#predefined-classes) or [mixins]({{ site.baseurl }}/layout/grid/#sass-mixins) to create horizontal forms. Add the `.row` class to form groups and use the `.col-*-*` classes to specify the width of your labels and controls.
```
Comment (mdo) [inline docs/components/forms.md]: I feel like the `{{ site.baseurl }}` instances in our Markdown aren't necessary... but maybe I'm thinking of something else? 🤔

**Review by mdo — CHANGES_REQUESTED** [review]
Thanks for this! Curious about some of the markup changes to the homepage though.

File: docs/components/forms.md (hunk 3 above)
Comment (bardiharborow) [inline docs/components/forms.md]: site.baseurl is `""` by default, but allows things link `preview.twbsapps.com` to work correctly. I'm just adding some that are missing.

File: docs/index.html (hunk 1 above)
Comment (bardiharborow) [inline docs/index.html]: The fragment is broken and the text sounded like it was related to the removed customiser, so I just removed it. I'll take this change out and let you decide what you want to do with it.

File: docs/index.html (hunk 2 above)
Comment (bardiharborow) [inline docs/index.html]: Once I removed the `Customized builds` section, these resulted in too much padding.

**Comment by bardiharborow** [comment]
@mdo, I've pushed a new commit which fixes those issues.

**Review by mdo — APPROVED** [review]
Thanks!

**Pull request merged**